pkg/cli/cmd/cli.go

```diff
@@ -43,12 +50,17 @@ type CLI struct {
 	NoCheck   bool         `name:"no-check" short:"N" env:"TOPAZ_NO_CHECK" help:"disable local container status check"`
 }
 
-func CheckRunning(c *cc.CommonCtx) error {
+func CheckRunning(c *cc.CommonCtx, containerName string) error {
```

Comment:
Can this be a private method, like grpcAvailable as it is only used on the command handlers and resides in the same package?